app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,50 @@
 class JobsController < ApplicationController
   def index
-    @jobs = jobs_payload
+    @jobs = Job.all
+    render json: {jobs: @jobs}
+  end
+
+  def my_jobs
+    @jobs = current_user.jobs + Job.where(provider_id: current_user.id)
+    render json: {jobs: @jobs}
   end
 
   def new
     render 'new.html.erb'
   end
 
+  def create
+    @job = Job.new(
+      requested_time: params[:requested_time],
+      user_id: current_user.id,
+      status: "posted"
```

Comment:
While this is the way we were all taught to set up a create action this is not the way we should set up the create action. This is not safe or secure. User can inject what ever they wanted into our code base.
>It provides an interface for protecting attributes from end-user assignment. This makes Action Controller >parameters forbidden to be used in Active Model mass assignment until they have been explicitly enumerated.
>In addition, parameters can be marked as required and flow through a predefined raise/rescue flow to end up as a >400 Bad Request with no effort.

Please change this to incorporate `StrongParameters`

#### Resources
[ruby docs](https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html)